parquet-fuzz/out/.gitignore

```diff
@@ -0,0 +1 @@
+*
```

Comment:
Maybe we just delete this file? Global gitignore should be applicable for subproject.